plat/xen/Config.uk

```diff
@@ -11,7 +11,6 @@ menuconfig PLAT_XEN
        select LIBNOLIBC if !HAVE_LIBC
        select HAVE_FDT if ARCH_ARM_64
        imply LIBFDT if (ARCH_ARM_32 || ARCH_ARM_64)
-       select XEN_DBGEMERGENCY if ARCH_ARM_32
```

Comment:
no mention about this change in the commit description.

Reply:
Good point. Actually, I should keep the line and only adopt it to the actually driver name and update it to use an `imply` instead of `select` (coming from re-arch considerations). We decided some time back to retire Xen/Arm32 support because it is unmaintained and outdated. I also do not have a machine anymore where I could test it. However, that should go to a different PR anyways. Do you have use and interest for Xen/Arm32?